config/settings/production.py

```diff
@@ -21,8 +21,14 @@
     # See:
     #   - https://github.com/django/channels_redis/issues/235
     #   - https://github.com/jazzband/django-redis/issues/353
-
-    CACHES["default"]["OPTIONS"]["CONNECTION_POOL_KWARGS"] = {"ssl_cert_reqs": False}
+    #   - https://paltman.com/how-to-turn-off-ssl-verify-django-rq-heroku-redis/
```

Comment:
The blog post references https://github.com/rq/django-rq/pull/509, which was merged and released on `django-rq` v2.5.0